client/js/components/week/week-header.js

```diff
@@ -4,8 +4,10 @@ var React = require('react');
 var WeekHeader = React.createClass({
   render: function () {
     return (
-      <div>
-        <h5>Week Header</h5>
+      <div className="date-bar">
+        <div className="arrow arrow-left"></div>
+        <h2>May 25 - May 30</h2>
```

Comment:
correct sir. will fill dynamically once the api calls are running. looking forward to it!